REFACTOR-BALANCE: Стабильная экономика - Пересмотр цен на многие вещи

KOCMODECAHTHUK opened the pull request:
## Описание / Что этот PR делает
Перерабатывает баланс карго, и переосмысляет карго.
Возрождает ПР #1573 - нормальный рефакторинг который никто не признал в свое время.
- Цены в карго: Теперь кратны `25`
- Меняет разброс цен:
  - У общего карго: от `1.0` до `1.2` 
  - У фракций: от `0.8` до `1.0` 
  - Патроны всегда имеют стабильную цену
  - Упрощает функцию на разброс цен у фракций
- Удаляет тонну устаревшего кода.
- Делает некоторые неоправданно дорогие вещи - более доступными.
- Не так сильно меняет экономику некоторых вещей, но значительно трогает все в целом

## Причина создания ПР / Почему это хорошо для игры
Баланс - Если так нужно, значит так нужно.
Изменения затрагивают `цены / наполненности / баланса` в карго и черного рынка.

Решает целый ряд, проблем и недоработок.
- Resolve #2434
- Resolve #2544
- Resolve #2585
- Resolve #2604
- Resolve #2605
- Resolve #2606
- Resolve #2607
- Resolve #2627
- Resolve #2628
- Resolve #2631

## Демонстрация изменений / Тестирование

<details><summary>Скриншоты/Видео с изменениями</summary>
<img width="393" height="669" alt="image" src="https://github.com/user-attachments/assets/c4231cf0-9870-4efa-aaae-0120d4da647d" /></details> 

## Список изменений

```yml
🆑
del: То что почистилось во время Рефакторинга
balance: Карго, цены, паки
/🆑
```

KOCMODECAHTHUK committed:
remove: bounties, light tweak price

Удаляем информацию баунти-наградах, они вообще то для карго-шаттла используются которого нету в этом билде лол.

MrCat15352 commented on mod_celadon/outpost_console/code/supply_pack/independent/misc/material.dm:
```diff
@@ -53,39 +53,46 @@
+/datum/supply_pack/faction/independent/material/sandbags50
+	name = "50 Sandbags"
+	desc = "It's so sand? Yes, it's sandbags."
+	cost = 1000
```
Почему мешки с песком стоят 1000, а вот ниже песочные блоки, готовый продукт 500, То есть сырье стоит в 2 раза дороже изготавливаемой продукции?

KOCMODECAHTHUK replied:
Мешков с песком много, надо бы уменьшить кол-во. М-г-м.

MrCat15352 commented:
А почему, зачем удалены разбросы цен у фракций? В том и смысл был, что за них ты мог получить скидку намного лучше чем у общего карго

KOCMODECAHTHUK committed:
until: More Rework prices

Меняет очень много цен, и удаляет лишний дублирующийся код

KOCMODECAHTHUK committed:
add: SELL ALL IT GUNS

Добавляет возможность продавать каждую пушку из карго в терминале на аванпосте, с поправкой на цену в 10 раз дешевле чем её можно купить

KOCMODECAHTHUK committed:
add: Sale faction and stable IND

Правит формулу в цене для общих товаров в карго
Возвращает скидки для фракций
Удаляет дефайны которые используются в одном файле, перемещая их туда с включением и отключеним там же.

KOCMODECAHTHUK committed:
tweak: Add work sea river more more more

Добавлены примерные формулы как считать цену продажи в коментарии для других кодеров на будущее
Изменены цены на миссии буров (Почему они вообще не были загружены, what?)

KOCMODECAHTHUK committed:
tweak: Добавлены стабитность в экономику патронов, никакой наценки на них

Поменял формулу в патронах, что бы они никогда не имели наценку и всегда были со стабильными ценами
Патроны нельзя продать как либо продать, они должны быть дешевыми
Исключение составляет лишь патроны на фракционное оружие, на них повышеные тарифы в общем карго если таковые есть либо вообще убрать от туда.

KOCMODECAHTHUK committed:
remove: Этот параметр скидки не учитывается в формуле

KOCMODECAHTHUK committed:
tweak: По мелочи убрано лишнее

KOCMODECAHTHUK committed:
tweak: Начало сортировки оружия

KOCMODECAHTHUK committed:
tweak: Сортировка паков в карго

KOCMODECAHTHUK committed:
spellcheck: Remove crate

KOCMODECAHTHUK committed:
tweak: Перетасовка названий паков

KOCMODECAHTHUK committed:
balance: Как же много всего

KOCMODECAHTHUK committed:
add: Shield - Disabler weakness

KOCMODECAHTHUK committed:
Merge commit beta-dev into cagro-balance

KOCMODECAHTHUK committed:
fix: update

KOCMODECAHTHUK committed:
tweak: Другой подход к фиксированным ценам

KOCMODECAHTHUK committed:
add: No random price - Misc and Food

KOCMODECAHTHUK committed:
fix: Super Sporter (7.62x40mm)

KOCMODECAHTHUK committed:
fix: Claris - Speedloader (FE pellet) - 22 rounds

KOCMODECAHTHUK committed:
tweak: Плазма к НТ, мелкие правки

KOCMODECAHTHUK committed:
fix: cm5_9mm caliber

KOCMODECAHTHUK committed:
mv: Plasma NT and Inteq

KOCMODECAHTHUK committed:
add: SolFed new guns

KOCMODECAHTHUK committed:
fix: Solfed gun category

KOCMODECAHTHUK committed:
remove: SF and Syndi Gun

KOCMODECAHTHUK committed:
spell-check: Мелкие опечатки

KOCMODECAHTHUK committed:
add: [TM] New Year Item

KOCMODECAHTHUK committed:
add: Ammo in cargo

KOCMODECAHTHUK committed:
fix: Ammo

KOCMODECAHTHUK committed:
tweak: Magazine remove absorde price

KOCMODECAHTHUK committed:
fix: Seed price

KOCMODECAHTHUK committed:
spellcheck: hades -> E-Assault-Rifle

KOCMODECAHTHUK committed:
tweak: MK SWAT 2 price

KOCMODECAHTHUK committed:
spellcheck: Asp - Auto-Pistol -> Pistol

KOCMODECAHTHUK committed:
fix: Plasma Cell price

KOCMODECAHTHUK committed:
add-remove: Много букв-анализа

KOCMODECAHTHUK committed:
add: Ashhand

KOCMODECAHTHUK committed:
remove: Plasma Weapon

KOCMODECAHTHUK committed:
add: inteq P46 Schnauzer - Magazine (4.6x30mm)

KOCMODECAHTHUK committed:
fix: Plasma Cell

KOCMODECAHTHUK committed:
fix: Plasma Weapon NT simple-mob

KOCMODECAHTHUK committed:
fix: NT -> IQ Magazine (4.6x30mm)

KOCMODECAHTHUK committed:
add: [TM] Radio-faction

KOCMODECAHTHUK committed:
Revert "fix: Plasma Weapon NT simple-mob"

This reverts commit 9d9ae1ca7a70d6931a4281649f7bf8db1e528abb.

KOCMODECAHTHUK committed:
fix: misstake

KOCMODECAHTHUK committed:
add: Пополнение к InteQ

KOCMODECAHTHUK committed:
Merge commit '9fb4e6dd7e5796f06ba82d80f936747db020ed72' into cagro-balance

KOCMODECAHTHUK committed:
add: Продажа всех подтипов и добавление Тайпана Синди

KOCMODECAHTHUK committed:
balance: E-SG 500 -> Bulky Weapon

KOCMODECAHTHUK committed:
fix: Polar-bear

KOCMODECAHTHUK committed:
fix: Inteq Magazines

KOCMODECAHTHUK committed:
fix: Solfed flag no heals

KOCMODECAHTHUK committed:
fix: Crusher-loot - crusher trophy

KOCMODECAHTHUK committed:
add: Sell more armor type

KOCMODECAHTHUK committed:
tweak: AI and Borg cargo

KOCMODECAHTHUK committed:
add: Inducer

KOCMODECAHTHUK committed:
Revert "balance: E-SG 500 -> Bulky Weapon"

This reverts commit 848b266de9b3352f15b376669b855c4d466e0ea1.

KOCMODECAHTHUK committed:
nerfed: Sell

KOCMODECAHTHUK committed:
fix: Crasher Trophy sell + nerf RCD

KOCMODECAHTHUK committed:
tweaks: anomaly cof + fix borgo price

KOCMODECAHTHUK committed:
fix: bug akita magazines

KOCMODECAHTHUK committed:
add: SSG-08 Saluki - Magazine

KOCMODECAHTHUK committed:
Merge branch 'beta-dev' into cagro-balance

KOCMODECAHTHUK committed:
fix: sell ore

KOCMODECAHTHUK committed:
fix: subtype gun sell

KOCMODECAHTHUK committed:
remove: sell tools

KOCMODECAHTHUK committed:
Merge branch 'beta-dev' into cagro-balance

KOCMODECAHTHUK committed:
remove: New Year item

KOCMODECAHTHUK committed:
Merge branch 'beta-dev' into cagro-balance

KOCMODECAHTHUK committed:
Merge branch 'beta-dev' into cagro-balance

KOCMODECAHTHUK committed:
fix: remove sell plasma-gas
